apps/dashboard/src/components/molecules/UserSearch/UserSearch.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState, type FC } from "react"
+import GenericSearch from "../../GenericSearch"
+import { useSearchUsersFromIDP } from "../../../modules/user/queries/use-user-get-query"
+import { User } from "@dotkomonline/types"
+
+interface UserSearchProps {
+  onSubmit(data: User): void
+}
+
+export const UserSearch: FC<UserSearchProps> = ({ onSubmit }) => {
+  const [searchQuery, setSearchQuery] = useState("")
+  const { usersFromIdp } = useSearchUsersFromIDP(searchQuery)
```

Comment:
Maybe debounce this so it doesn't send a request each time the text is changed? Can definitely fix in a separate PR though!